### DISCUSSION TITLE
Исчезает полоса прокрутки

### DISCUSSION THREAD

**Discussion by Killerbl** [Chimildic/YaMuTools#34] [Вопрос/ответ] [answered]
Расширение удаляет вертикальную полосу прокрутки со страницы плейлиста, если это не ошибка, а функция - то хотелось бы иметь опцию её отключения

**Comment by Pushkin31** [comment]
Поддерживаю. В chrome`образных браузерах при переходе в сам альбом - пропадает скролл в браузере. В мозилле всё отлично.

**Comment by Chimildic** [comment, marked as answer]
Исправлено в версии 0.8.0
